app/assets/javascripts/models/user.js

```diff
@@ -1,3 +1,15 @@
 module.exports = Backbone.Model.extend({
-  name: 'user'
+  name: 'user',
+  initialize: function() {
```

Comment:
If I don't add the constructor it will return me `Uncaught TypeError: Cannot read property 'hasOwnProperty' of undefined` when I try to access the `getCurrent` function.